src/toolboxes/paintbrush/Canvas.tsx

```diff
@@ -235,7 +235,7 @@ export class CanvasClass extends Component<Props, State> {
       context.strokeStyle = brush.color;
       this.annotationOpacity = this.props.annotationAlpha;
     }
-    if (isActive || brush.type === "erase") {
```

Comment:
I think `isActive` here was added to apply the eraser only on the active annotation.